v2/src/components/recipeBoxes/guides.json

```diff
@@ -0,0 +1,79 @@
+{
+  "auth_modes": [
+    {
+      "title": "Email password Login",
+      "icon": "emailPass",
+      "path": "/docs/emailpassword/introduction",
+      "img": "emailPassSS"
+    },
+    {
+      "title": "Social Login",
+      "icon": "social",
+      "path": "/docs/thirdparty/introduction",
+      "img": "socialSS"
+    },
+    {
+      "title": "Email password + Social Login",
+      "icon": "emailPassPlusSocial",
```

Comment:
The inner shadow in the hover state is not as per designs

<img width="241" alt="Screenshot 2023-03-13 at 6 58 57 PM" src="https://user-images.githubusercontent.com/18233774/224715883-127513d5-6c38-4b75-bd02-716ad507096b.png">

Reply:
This is still pending